build.gradle

```diff
@@ -9,6 +9,9 @@ subprojects {
 
     task wrapper(type: Wrapper) {
         gradleVersion = '6.2'
+        onlyIf {
```

Comment:
@ErikSchierboom since I can't find an easy way to ignore some folders from listing (commented [here](https://github.com/exercism/configlet/issues/196)):
* I've removed `gradle` folder and wrappers from "exercises" and "exercises/practice" folders (they were of no use anyhow).
* I've configured the Gradle wrapper task to only generate wrapper files for subprojects that have `build.gradle` file.